Integrations/FeedOffice365/FeedOffice365.yml

```diff
@@ -112,30 +105,31 @@ script:
       secret: false
     - auto: PREDEFINED
       default: false
-      defaultValue: 'IPS'
-      description: The indicator type. Can be "IPS", "URLS", or "Both". The default value is "IPS".
+      defaultValue: IPS
+      description: The indicator type. Can be "IPS", "URLS", or "Both". The default
```

Comment:
This is a little weird now, isn't it? Because you're asking them to choose based off of how O365 divvies up their indicators, but in reality, `IPS` contains `IPv4`, `IPv6`, `CIDR`, and `IPv6CIDR` and `URLS` contains `URL` and `Domain` indicator types so we might want to change this filter to be for how we return indicators. Also, please change `URLS` to `URLs` and `IPS` to `IPs`, in all the relevant fields of this parameter as well as in your code.